charts/rancher-turtles/100.1.0+up0.5.0/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher Turtles - the Cluster API operator
+  catalog.cattle.io/kube-version: '>= 1.23.0-0'
```

Comment:
Please se the following Rancher Version Annotations rules at: https://github.com/rancher/charts/blob/dev-v2.8/README.md#rancher-version-annotations